app/models/dataset_file.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class DatasetFile
+  attr_accessor :name, :description, :format, :size, :mime_type, :sequence, :handle
+
+  def self.from_hash(data)
+    hash = data.with_indifferent_access
+    file = DatasetFile.new
+    file.name = hash[:name]
+    file.description = hash[:description]
+    file.mime_type = hash[:mime_type]
+    file.size = hash[:size]
+    file.mime_type = hash[:mime_type]
+    file.sequence = (hash[:sequence] || "").to_i
+    # Technically the handle is a property of the dataset item rather than the file (aka bitstream)
+    # but we store it at the file level for convenience.
+    file.handle = hash[:handle]
+    file
+  end
+
+  def self.download_root
+    "https://dataspace-dev.princeton.edu/bitstream"
```

Comment:
Shouldn't this be in config, since it'll vary between environments?

Reply:
@escowles yes, we have a separate ticket to handle the different environments so I used the same hard coded value that we are currently using for the indexing.

Reply:
@hectorcorrea 👍🏻 OK, great — that sounds reasonable to me!